docs/installation.rst

```diff
@@ -18,8 +18,18 @@ Install from PyPi
 
 .. code-block:: shell
 
-   $ pip install ansible-builder
+   $ pip3 install ansible-builder
 
+.. note::
+
+   An **alternative** approach to install ``ansible-builder`` is using the ``ansible-dev-tools`` package.
+   `Ansible Development Tools (ADT) <https://ansible.readthedocs.io/projects/dev-tools/>`_ is a single Python package that includes all necessary tools to
+   set up a development environment, generate new collections, build and test the content consistently, resulting in robust automation.
```

Comment:
```suggestion
   set up a development environment, generate new collections, build and test the content.
```

Reply:
In the current form it's not clear if "consistently" modifies "build and test the content" or if it also modifies "set up a development environment" and "generate new collections".

If the former, then "consistently build and test the content". If the latter, then "consistently set up a development environment, generated new collections, build and test the content."

Also ", resulting in robust automation" looks like a dangling participle. If you want to keep it, maybe consider ", which results in robust automation."